### ISSUE TITLE
Feature Request

### ISSUE THREAD

**Issue by GothkittyGirl** [ChestShop-authors/ChestShop-3#153]
[Please search for tickets with the same issues as yours first before reporting!]

#### Plugin Version
[3.9.3-snapshot (build 89]

Please provide any additional info below
wondering if it would possible to allow for multiply items to a shop?

**Comment by Phoenix616** [comment]
The biggest hurdle for such a feature is the fact that a sign has a very limited amount of space where information about the shop's config can be stored. The only really implementable way would be to allow wildcard shops that don't care about the item and just blindly/randomly sell/buy items from the chest for the same price.

**Comment by GothkittyGirl** [comment]
ahh ok. only reason i asked is because a plugin i used to use allowed this but had been abandoned since 1.13 and thought i would if it was possible with this

**Comment by Nodws** [comment]
Yes please add  asterisk so we can use it as [random]  slot selector

**Comment by Phoenix616** [comment]
Another possibility for implementing this (maybe as an option): Allow the buyer to select which item to buy via a chest GUI.

**Comment by rcmaehl** [comment]
Looking for this feature as I have multiple of the same item with different NBT data that I'd like to sell. Ideally, I'd like to see like Iron_Ingot#* for any Iron_Ingot with any NBT data without showing the players what the NBT data is (e.g. for a lottery), and "[Selectable]" as the item type to allow players to pick certain items (for non-lotteries)
